### DISCUSSION TITLE
Caldera Login issue

### DISCUSSION THREAD

**Discussion by Unit88** [mitre/caldera#3080] [Q&A]
I have deployed Caldera using the docker by following the guide here.
git clone https://github.com/mitre/caldera.git --recursive
All systems ready and I see this warning, 
I also edited the .env file under the magma folder and provided the local IP of the vm. VITE_CALDERA_URL=http://mylocalip:8888
After running the docker, I am getting the login page, trying the credentials admin admin / red admin, nothing happens and I cannot login, there is no error.  
![error](https://github.com/user-attachments/assets/d3868c1e-2b21-4d9d-9c4c-35de0851d5fc)
![pag](https://github.com/user-attachments/assets/3389d275-19f9-4629-b663-a78db5d47e1e)

**Comment by rfulwell** [comment]
Have you tried using the generated creds in your case?
https://caldera.readthedocs.io/en/latest/Installing-Caldera.html#installation
> Once started, log in to http://localhost:8888 with the red using the password found in the conf/local.yml file (this file will be generated on server start).

I believe the default creds will only work if you specify `--insecure` on the server startup command line:
https://caldera.readthedocs.io/en/latest/Server-Configuration.html

**Reply by FeelingApathy** [reply]
Hi rfulwell, I tried this, and with the --log DEBUG, here is the output when attempting to login:
![Image 1-2-25 at 9 40 AM](https://github.com/user-attachments/assets/b98371a7-5809-477f-b609-6efeebd56675)

the log in page sits and does nothing

**Comment by josh-thurston** [comment]
Same issue here.  Docker and building on my own Ubuntu server.  I can never get the authentication to work.  I have tried running --insecure with the default.yml credentials,  i have tried setting my own credentials.  nothing works.  I have to wonder why nobody has solved this and there hasn't been a release in about a year.  Did this project get abandoned?

**Comment by robin-wei** [comment]
Same issue here. After I did a lot of troubleshooting for the installation, I could run  "python3 server.py --insecure --build" and got the "All systems ready", but when I tried to logon use admin/admin, red/admin, I could never get the authentication to work as well. Can anyone help? Version is 5.0.0

**Comment by ash-al1** [comment]
I had this issue at first; open up the dev console in your browser and you'll see the unauth requests.

**Reply by robin-wei** [reply]
Thanks, so how did you solve it?

**Comment by shiracamus** [comment]
You have to use the auto-generated password in the docker container.

```
$ docker ps
CONTAINER ID   IMAGE            COMMAND               CREATED       STATUS       PORTS                                                                                                               NAMES
*** find the CONTAINER ID for your docker container running Caldera in the output ***

$ docker container exec YOUR_CALDERA_CONTAINER_ID cat conf/local.yml
ability_refresh: 60
api_key_blue: vRaRVoEcx2yWo4lXJQ653BoO3ZVKgMQeIch6x...
api_key_red: QEM0KH9kiIJiNsMvPjjmSjMKKV5nrkcUU2Edf...
app.contact.dns.domain: mycaldera.caldera
...
users:
  blue:
    blue: rMFvFflxKEW0qt3V0htX2acdTGEUdniE...
  red:
    red: -hQkhOZ2fFuSMXJU8Ax59VbQf9J3FUCbf...
```

In the above output example, -hQkhOZ2fFuSMXJU8Ax59VbQf9J3FUCbf... is the password for the red.
